app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -490,18 +488,6 @@ static void compareSecondaryIndices(String tableName, List<?> indices1, List<?>
                             " is changing the projection.");
                 }
 
-                if (!globalIndex1.getProvisionedThroughput().getReadCapacityUnits().equals(
-                        globalIndex2.getProvisionedThroughput().getReadCapacityUnits())) {
-                    throw new BridgeInitializationException("Table " + tableName + " global index " + indexName +
-                            " is changing the provisioned read capacity.");
-                }
```

Comment:
If we need to meet about this, let's meet about this. Whatever error you are seeking to prevent here, you are also preventing us from reacting in real time to scaling challenges.